client/blocks/author-selector/switcher-shell.jsx

```diff
@@ -46,23 +40,14 @@ class AuthorSwitcherShell extends React.Component {
 	authorSelectorToggleRef = createRef();
 	authorSelectorChevronRef = createRef();
 
-	UNSAFE_componentWillMount() {
-		this.instance = instance;
-		instance++;
-	}
-
 	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if (
-			! nextProps.fetchOptions.siteId ||
-			nextProps.fetchOptions.siteId !== this.props.fetchOptions.siteId
-		) {
+		if ( ! nextProps.siteId || nextProps.siteId !== this.props.siteId ) {
 			this.props.updateSearch( false );
```

Comment:
In this component, `updateSearch` is called multiple times with `false`. However, this will lead to an error, because in `AuthorSelector` we're attempting to `trim()` the value:

![](https://cldup.com/sgJPc76zvY.png)

Seems like it should be pretty straightforward and painless to update the `false`  instances to an empty string`''`.

Reply:
Great catch, thanks @tyxla! I'll make sure to fix this 👍🏻